main.cc

```diff
@@ -1159,6 +1159,7 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
                 return sys_ks.start(snitch.local());
             }).get();
             cfg->host_id = sys_ks.local().load_local_host_id().get0();
+            utils::fb_utilities::set_host_id(cfg->host_id);
```

Comment:
I really wish we didn't pile onto this `fb_utilities` hack, but we can fix it another time.